src/lib/backend/Memory.js

```diff
@@ -311,6 +313,18 @@ export default class Memory {
     }
 
     /**
+    * publish a message to a channel, clients can subsribe to this channel
+    * to receive messages
+    * @param {string} channel - channel to push the message
+    * @param {string} message - message to be pushed to the channel
+    * @return {object} - current instance
+    */
+    publish(channel, message) {
+        this.channels.emit('utapi', message);
```

Comment:
I wrote this memory backend to emulate Redis. The `ioredis` module sends two vars in the callback, and for me to access the second param I have to include the first param as well. I am also thinking of changing this channel name but you are right. That was a typo, channel should be used as param.